packages/mocktail/pubspec.yaml

```
@@ -5,7 +5,7 @@ repository: https://github.com/felangel/mocktail
 homepage: https://github.com/felangel/mocktail/tree/main/packages/mocktail
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.18.0 <3.0.0"
```

```suggestion
  sdk: ">=2.12.0 <3.0.0"
```

Can we revert this since it shouldn't be necessary for this PR?